dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -7,23 +7,16 @@ import 'sentry_options.dart';
 
 /// converts [StackTrace] to [SentryStackFrames]
 class SentryStackTraceFactory {
-  SentryOptions _options;
+  SentryOptions options;
 
   final _absRegex = RegExp('abs +([A-Fa-f0-9]+)');
   static const _stackTraceViolateDartStandard =
       'This VM has been configured to produce stack traces that violate the Dart standard.';
 
-  SentryStackTraceFactory(SentryOptions options) {
-    if (options == null) {
-      throw ArgumentError('SentryOptions is required.');
-    }
-    _options = options;
-  }
+  SentryStackTraceFactory(this.options);
 
   /// returns the [SentryStackFrame] list from a stackTrace ([StackTrace] or [String])
   List<SentryStackFrame> getStackFrames(dynamic stackTrace) {
-    if (stackTrace == null) return null;
```

Comment:
This method should probably be changed to `getStackFramesFromString(String)` and `getStackFramesFromString(StackTrace)` because all other types and null are not allowed.

Reply:
yep, but the docs mention that and we check its types in the method body, its an internal class, I'd say we are fine, nobody should use it directly